src/account/MyAccount/MyPassword.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from "react";
+import Card from "../Card";
+import CardHeader from "../CardHeader";
+import MyPasswordContent from "./MyPasswordContent";
+import MyPasswordEdit from "./MyPasswordEdit";
+
+const MyPassword: React.FC = () => {
+  const [isEditing, setEditing] = useState(false);
+  const onClick = () => {
+    setEditing(!isEditing);
+  };
+  const header = (
+    <CardHeader title="MY PASSWORD" editing={isEditing} onClick={onClick} />
```

Comment:
Wouldn't that be better to have the property `text-transform: uppercase` instead of manually doing that?